### PR TITLE
Ericscript recipe

### PR DESCRIPTION
**Pull request by tetianakh**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

This is a recipe for EricScript, a tool for gene fusion calling: https://sites.google.com/site/bioericscript/
@bioconda/core

### CODE REVIEW

File: recipes/ericscript/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package:
+  name: ericscript
+  version: 0.5.5
+
+source:
+  fn: ericscript-0.5.5.tar.bz2
+  url: https://sourceforge.net/projects/ericscript/files/ericscript-0.5.5.tar.bz2
```
Comment (tomkinsc) [inline recipes/ericscript/meta.yaml]: Please add a [hash](http://conda.pydata.org/docs/building/meta-yaml.html#source-from-tarball) for the archive file, so conda build will check its integrity. An md5 hash is fine, but sha1 and sha256 are also supported.

Comment (tetianakh) [inline recipes/ericscript/meta.yaml]: @tomkinsc  added md5 checksum.

Comment (tetianakh) [inline recipes/ericscript/meta.yaml]: @tomkinsc should I squash the commits?

Comment (tomkinsc) [inline recipes/ericscript/meta.yaml]: Yes, please.

Comment (bgruening) [inline recipes/ericscript/meta.yaml]: We can do this for you during merge - github feature! :)

Comment (tetianakh) [inline recipes/ericscript/meta.yaml]: @bgruening mind blown. I've had no idea! :laughing:

Comment (bgruening) [inline recipes/ericscript/meta.yaml]: But it is still failing.

**Comment by tetianakh** [comment]
The CI build fails due to timeout:
```
/home/travis/build.sh: line 151:  3410 Terminated              travis_jigger $! $timeout $cmd
The command "travis_wait 120 scripts/travis-run.sh" exited with 1.
```
I can build the recipe on CentOS 6 locally. Any ideas what is breaking the build?

**Comment by bgruening** [comment]
What happens with simulate_travis.py? My guess is that you are missing some runtime dependencies and the testing hangs at some point.

**Comment by tetianakh** [comment]
@bgruening I get the exact same problem as described in the issue #3126.

**Comment by bgruening** [comment]
can you change the `travis_wait 120` in the travis.yaml file to `travis_wait 60` and see what happens?

**Comment by tetianakh** [comment]
@bgruening everything is fixed!

**Comment by bgruening** [comment]
Great! Thanks a lot!

**Pull request merged**